scripts/cnv_wdl/cnv_common_tasks.wdl

```diff
@@ -605,3 +616,122 @@ task CollectModelQualityMetrics {
         String qc_status_string = read_string("qcStatus.txt")
     }
 }
+
+task BundlePostprocessingInvariants {
+    input {
+      Array[File] calls_tars
+      Array[File] model_tars
+      Array[File] calling_configs
+      Array[File] denoising_configs
+      Array[File] gcnvkernel_version
+      Array[File] sharded_interval_lists
+
+      # Runtime parameters
+      String docker
+      Int? mem_gb
+      Int? disk_space_gb
+      Boolean use_ssd = false
+      Int? cpu
+      Int? preemptible_attempts
+    }
+
+    command <<<
+        set -euo pipefail
+        mkdir -p out
+
+        calls_files_tar_list=~{write_lines(calls_tars)}
+        model_files_tar_list=~{write_lines(model_tars)}
+
+        calling_configs_list=~{write_lines(calling_configs)}
+        denoising_configs_list=~{write_lines(denoising_configs)}
+        gcnvkernel_version_list=~{write_lines(gcnvkernel_version)}
+        sharded_interval_lists_list=~{write_lines(sharded_interval_lists)}
+
+        cat $calls_files_tar_list | sort -V > calls_files_tar_list.sorted
+        cat $model_files_tar_list | sort -V > model_files_tar_list.sorted
+
+        cat $calling_configs_list | sort -V > calling_configs_list.sorted
+        cat $denoising_configs_list | sort -V > denoising_configs_list.sorted
+        cat $gcnvkernel_version_list | sort -V > gcnvkernel_version_list.sorted
+        cat $sharded_interval_lists_list | sort -V > sharded_interval_lists_list.sorted
```

Comment:
Hm not sure what was happening there, especially since my own tests completed successfully. But my latest changes don't seem to be triggering this error.